index.html

```diff
@@ -25,11 +25,11 @@
     <header class="profile">
```

Comment:
_This request is for  MY RECENT WORK section, since you haven't made any changes there I can't put my comment right there_
- [ ] Make changes in your existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

Reply:
Please can you elaborate more.